rules/aws_cloudtrail_rules/aws_ec2_ebs_encryption_disabled.py

```diff
@@ -0,0 +1,16 @@
+from panther_base_helpers import aws_rule_context
+
+
+def rule(event):
+    return (
+        event.get("eventSource") == "ec2.amazonaws.com"
+        and event.get("eventName") == "DisableEbsEncryptionByDefault"
+    )
+
+
+def title(event):
+    return "EC2 EBS Default Encryption was disabled in " f"[{event.get('recipientAccountId')}]"
```

Comment:
this is a regional setting ( or was ~1 year back ). I don't remember offhand if region's in the alert_context, and it could be helpful to include it in the title.